Xabe.FFmpeg/FFmpeg.cs

```diff
@@ -228,20 +228,20 @@ private void FindProgramsFromPath(string path)
             process.Start();
             FFmpegProcessId = process.Id;
 
-            if (priority.HasValue)
+            try
             {
-                process.PriorityClass = priority.Value;
-            }
-            else
-            {
-                try
+                if (priority.HasValue)
                 {
-                    process.PriorityClass = Process.GetCurrentProcess().PriorityClass;
+                    process.PriorityClass = priority.Value;
                 }
-                catch (Win32Exception e) when (e.Message.Contains("Permission denied "))
+                else
                 {
+                    process.PriorityClass = Process.GetCurrentProcess().PriorityClass;
                 }
             }
+            catch (Exception e)
```

Comment:
Please remove assigning exception to variable if you don't use this variable later.
Restrictions about builds are very tight and because of that build on Travis doesn't pass.
(just write "catch (Exception)")